subjects/java/checkpoints/double-linkedlist/README.md

```diff
@@ -0,0 +1,144 @@
+## Double Linked List
+
+### Instructions
+
+Create a doubly linked list data structure that implements the following methods:
+
+- `at(int index)`: to access an element by its index.
```

Comment:
What should happen with negative or, more general, out of bounds indexes?

Reply:
same as the single linked list, they have to return -1